prod.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use Nvidia's latest Ubuntu 22.04 image as the base image
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
+
+LABEL org.opencontainers.image.source https://github.com/khoj-ai/khoj
+
+# Install System Dependencies
+RUN apt update -y && apt -y install python3-pip git
+
+WORKDIR /app
+
+# Install Application
+COPY pyproject.toml .
+COPY README.md .
+RUN sed -i 's/dynamic = \["version"\]/version = "0.0.0"/' pyproject.toml && \
+    TMPDIR=/home/cache/ pip install --cache-dir=/home/cache/ -e .
+
+# Copy Source Code
+COPY . .
+
+RUN apt install vim -y
```

Comment:
Is vim install required here?

Reply:
Always useful for debugging, I figured we may as well pre-install it to make it easier to maintain.